packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,377 @@
+
+1. [Authenticate with Okta Aerial](#authenticate-with-okta-aerial): Configure an API client in the Aerial org to call the Okta Aerial API.
+1. [Create and configure a child org](#create-a-child-org): Create a child org based on the parent org using the Org creator API.
+1. [Link the org to the Aerial account](#link-the-org-to-the-aerial-account): Link the org to the Aerial account using the Aerial API.
+1. [Enable products in the org](#enable-products-in-the-org): Enable products in the Org using the Aerial API.
+1. [Configure the org](#configure-the-org): Configure the org using Okta objects in the Aerial API.
+
+You can also manage an org's status, and enable or disable products in an org:
+
+- [Manage an org's status](#manage-an-orgs-status)
+- [List the configured products in the Org](#list-the-configured-products)
+- [Enable a product in an Org](#enable-a-product-in-an-org)
+- [Remove a product from an Org](#remove-a-product-from-an-org)
+
+<div class="full">
+
+![Okta Aerial - summary of steps](/img/aerial-full-flow.png)
+
+</div>
+
+## Register an Okta Aerial API client
+
+Use an Aerial API client to add orgs to the Aerial account and to modify products.
+
+Only a super admin in your Aerial org can grant scopes to the client. Configure scopes once per API client.
+
+In your Aerial org, create an API client:
+
+1. In the **Admin Console**, go to **Applications** > **Applications**, and then click **Create App Integration**.
+1. Select **API Services** as the sign-in method, and click **Next**.
+1. Enter a name for your client and click **Save**.
+
+Configure the signing keys for the client:
+
+1. In the **Client Credentials** section of the **General** tab, click **Edit** to change the client authentication method.
+1. Select **Public key/Private key** as the client authentication method.
+1. Choose either **Save keys in Okta** or **Use a URL to fetch keys dynamically**:
+   - If you want to save keys in Okta, click **Add key**.
+   - If you want to use a URL to fetch keys dynamically, you need to provide a URL that returns the [JWKS documents](https://www.rfc-editor.org/rfc/rfc7517#section-5).
+
+Grant scopes to the client:
+
+1. Select the **Okta API Scopes** tab.
+1. To access Okta Aerial, click **Grant** on the following scopes:
+  - `okta.accounts.manage`: read/write operations
+  - `okta.accounts.read`: read-only operations
+
+## Authenticate with Okta Aerial
+
+To authenticate with Okta Aerial, a client obtains an access token from the Aerial org. See [Implement OAuth for Okta with a service app](/docs/guides/implement-oauth-for-okta-serviceapp/main/#get-an-access-token) for more information.
+
+Create a [JWT assertion](/docs/guides/implement-oauth-for-okta-serviceapp/main/#create-and-sign-the-jwt) and use it to make a [token request](/docs/guides/implement-oauth-for-okta-serviceapp/main/#create-and-sign-the-jwt) to the Aerial org. The Aerial org returns the access token:
```

Comment:
The last sentence is confusing since you then present the request, not the response. I would move this last sentence below the token request example and above the "Add the access token to the Authorization header of Okta Aerial API requests" sentence.